webview/src/experiments/components/table/Indicators.tsx

```diff
@@ -1,42 +1,20 @@
-import React, { MouseEventHandler, ReactNode } from 'react'
+import React, { MouseEventHandler, ReactElement, ReactNode } from 'react'
 import { useSelector } from 'react-redux'
 import cx from 'classnames'
 import { MessageFromWebviewType } from 'dvc/src/webview/contract'
 import { FilteredCounts } from 'dvc/src/experiments/model/filterBy/collect'
-import { TippyProps } from '@tippyjs/react'
 import styles from './styles.module.scss'
+import { CellHintTooltip } from './CellHintTooltip'
 import { Icon } from '../../../shared/components/Icon'
 import {
   Filter,
   GraphScatter,
   SortPrecedence
 } from '../../../shared/components/icons'
 import { sendMessage } from '../../../shared/vscode'
-import Tooltip from '../../../shared/components/tooltip/Tooltip'
 import { pluralize } from '../../../util/strings'
 import { ExperimentsState } from '../../store'
 
-export type IndicatorTooltipProps = Pick<TippyProps, 'children'> & {
-  tooltipContent: ReactNode
-}
-
-export const IndicatorTooltip: React.FC<IndicatorTooltipProps> = ({
```

Comment:
Went ahead and deleted this component since we aren't using it anywhere.

Reply:
Thanks!